specification/ai/Face/models.session.tsp

```diff
@@ -157,15 +180,12 @@ model LivenessOutputsTarget {
   imageType: ImageType;
 }
 
-#suppress "@azure-tools/typespec-azure-core/documentation-required" "Values are self-documetning."
-#suppress "@azure-tools/typespec-azure-core/documentation-required" "Values are self-documetning."
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "Values are self-documenting."
 @doc("The model version used for liveness classification.")
 union LivenessModel {
   string,
-  v2020_02_15_preview_01: "2020-02-15-preview.01",
-  v2021_11_12_preview_03: "2021-11-12-preview.03",
   v2022_10_15_preview_04: "2022-10-15-preview.04",
-  v2023_03_02_preview_05: "2023-03-02-preview.05",
+  v2023_12_20_preview_06: "2023-12-20-preview.06",
 }
```

Comment:
Are these enum value to be versioned? Or the change to them applies across 1.1 and 1.2?

Reply:
Applies across 1.1 and 1.2.
Removed value are some internal model should not be seen by customer.
Added value is something missed in previous implementation.
Since `LivenessModel` only used by response in 1.1, I think it's okay to correct the actual value directly.
In 1.2, it will also be used in request.